fundbot/plugins/fund/data_source.py

```diff
@@ -2,9 +2,11 @@
 from functools import wraps
 from typing import Union, List, Dict
 import traceback
+import asyncio
 
 import httpx
 from nonebot.log import logger, default_format
+from bs4 import BeautifulSoup
```

Comment:
should modify `requirements.txt` when import new package.

Reply:
plz refer https://github.com/CommonProsperity/fundbot/blob/feature/time_range/requirements.txt#L4

Reply:
sorry, I didn't aware that.